### PR TITLE
Np 45757 fixing expansion

### PR DESCRIPTION
**Pull request by truhacevkir**
When `publication.entityDescription.contributor.affiliation` has labels
and affiliation is topLevelOrg
and labels for language value to topLevelOrg differ from label stored in database
new label is being appended to existing label:

So instead of creating indexDocument with following label:
```
"labels": {
      "el": "label from response"
    }
```
    
We persist following:

```
"labels": {
      "el": ["label from response", "label from database"]
    }

```

Which can't be serialized to Publication and causing failing test.

### CODE REVIEW

File: expansion/src/test/java/no/unit/nva/expansion/model/ExpandedDataEntryTest.java [hunk 1]
```diff
@@ -163,9 +162,24 @@ void shouldReturnExpandedResourceWithoutLossOfInformation(Class<?> instanceType)
...
+            .forEach(contributor -> contributor.get("affiliations")
+                                        .forEach(affiliation -> ((ObjectNode) affiliation).remove("labels")));
+
+        actualTree.get("entityDescription").get("contributors").forEach(contributor ->
+                                                                            contributor.get("affiliations").forEach(affiliation ->
+                                                                                                                        ((ObjectNode) affiliation).remove("labels")));
+
+        // Convert the trees back to JSON strings
+        jsonOriginal = objectMapper.writeValueAsString(originalTree);
+        jsonActual = objectMapper.writeValueAsString(actualTree);
```
Comment (truhacevkir) [inline expansion/src/test/java/no/unit/nva/expansion/model/ExpandedDataEntryTest.java:182]: This one will be refactored once we agree this is a solution.

File: expansion/src/main/java/no/unit/nva/expansion/model/IndexDocumentWrapperLinkedData.java [hunk 2]
```diff
@@ -56,14 +56,14 @@ public IndexDocumentWrapperLinkedData(UriRetriever uriRetriever) {
...
-        return new FramedJsonGenerator(inputStreams, frame).getFramedJson();
+        return new FramedJsonGenerator(indexDocument, inputStreams, frame).getFramedJson();
     }
 
     //TODO: parallelize
 
     private List<InputStream> getInputStreams(JsonNode indexDocument) {
         final List<InputStream> inputStreams = new ArrayList<>();
-        inputStreams.add(stringToStream(toJsonString(indexDocument)));
+//        inputStreams.add(stringToStream(toJsonString(indexDocument)));
```
Comment (monaullah) [inline expansion/src/main/java/no/unit/nva/expansion/model/IndexDocumentWrapperLinkedData.java:66]: Are you sure you want to remove this line? Is this not the persisted Publication? Not sure if test coverage in expansion is good enough to pick up if this is removing important data in the graph?

Comment (truhacevkir) [reply expansion/src/main/java/no/unit/nva/expansion/model/IndexDocumentWrapperLinkedData.java:66]: Yes, we know read indexDocument and remove labels here:

```
public FramedJsonGenerator(JsonNode indexDocument, List<InputStream> streams, String frame) {
        removeAffiliationLabels(indexDocument);
        var indexDocumentInputStream = stringToStream(toJsonString(indexDocument));
        streams.add(indexDocumentInputStream);
        var model = createModel(streams);
        framedJson = frameJsonLd(model, frame);
    }
```

File: expansion/src/test/java/no/unit/nva/publication/indexing/ExpandedResourceTest.java [hunk 3]
```diff
@@ -197,15 +200,14 @@ void shouldReturnIndexDocumentWithTopLevelOrganizationWithoutHasPartsIfContribut
         assertThat(deepestNestedSubUnit.at(JSON_PTR_ID).textValue(), is(equalTo(affiliationToBeExpanded.toString())));
     }
 
-    @Test
+    @RepeatedTest(100)
```
Comment (monaullah) [inline expansion/src/test/java/no/unit/nva/publication/indexing/ExpandedResourceTest.java:203]: `@Test` instead of `@RepeatedTest(100)`?

**Pull request closed without merging**